src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -151,7 +151,7 @@ private void EnableTimer(float pollingIntervalInSeconds)
                     {
                         s_pollingThreadSleepEvent = new AutoResetEvent(false);
                         s_counterGroupEnabledList = new List<CounterGroup>();
-                        s_pollingThread = new Thread(PollForValues) { IsBackground = true };
+                        s_pollingThread = new Thread(PollForValues) { IsBackground = true, Name = ".NET Counter Polling"  };
```

Comment:
In general do we name threads for what they are or what they're doing (e.g. ".NET Counter Poller" vs ".NET Counter Polling")?  I don't have a strong opinion, as long as we're consistent.